packages/testing/src/web/StorybookProvider.tsx

```diff
@@ -5,6 +5,7 @@ import { MockProviders } from './MockProviders'
 import { setupRequestHandlers, startMSW, mockCurrentUser } from './mockRequests'
 
 export const MockingLoader = async () => {
+  // @ts-expect-error not sure why this is happening yet
```

Comment:
Just noting that I need to fix this

Reply:
Done in https://github.com/redwoodjs/redwood/pull/8342/commits/6ec65845bf09c64bdc8fa3737ae6e84ee8dad050